### ISSUE TITLE
Command does not terminate with bigger inputs

### ISSUE THREAD

**Issue by philipgiuliani** [jayjun/rambo#1]
Hi,
when trying to pass a huge value as `stdin` to Rambo the command never terminates.

Steps to reproduce:
1. Download this sample text file (Its only 280kb of lorem ipsum...): [test.txt](https://github.com/jayjun/rambo/files/4205311/test.txt)
2. `{:ok, txt} = File.read("test.txt")`
3. `Rambo.run("cat", [], in: txt)`

With smaller files the command will return, but only a random amount of the input gets returned to the output.

**Comment by jayjun** [comment]
I can reproduce with any binary larger than 147,456 bytes.

```elixir
Rambo.run("cat", in: String.duplicate("?", 147456)) # works
Rambo.run("cat", in: String.duplicate("?", 147457)) # hangs
```

The default standard I/O buffer size is 4,096 bytes, and 4,096 × 36 is exactly 147,456.

I haven’t found where this limit comes from, but I know reading is blocked on the Rust side beyond this size.

**Comment by jayjun** [comment]
Thanks for the report! Fixed in 0.3.0.
